### ISSUE TITLE
help

### ISSUE THREAD

**Issue by Lu7z-727** [Cocofeat/TBraTS#7]
I'm sorry to bother you. I saw your question on the GitHub board on Focusnetv2. Have you solved the problem? Can you give me some help?

**Comment by Cocofeat** [comment]
Yeah, it may be solved. What's your question about the focusnetv2?

**Comment by Lu7z-727** [comment]
我卡在了SOL-Net。论文中小器官的SOLNet的平均定位误差（估计的质心和地面真实质心之间的平均距离）约为2.7mm。但是我使用文章源码无法达到这么高的精度。我训练的SOL-Net小器官最好结果9mm左右，具体大概为视交叉神经18  其他两个5 6 左右。 这是不是会导致对抗性训练small dice为0的情况？  请问你是用提供的源码进行训练的嘛？ 感觉很奇怪。

**Comment by Cocofeat** [comment]
I have used his source code to my own task, but did not follow up his source code to reproduce his results. I pay more attention to the heatmap in SOL. But I guess your problem here may be caused by the prediction of the small organ input to calculate the dice. If there is an error, the dice will not be 0. 
Sorry for not being able to solve your problem. The author may pay little attention to the page of the github due to time constraints, maybe you can email him.
